### DISCUSSION TITLE
Mesh generation filled with holes

### DISCUSSION THREAD

**Discussion by moosyah** [NVlabs/instant-ngp#1482] [Q&A]
Hi, 

I've been trying to create a 3D mesh of my car. In the viewer the render looks great, but as soon as I try to generate a mesh, it gets filled with these holes like a swiss cheese. What am I doing wrong?

![Screenshot_2](https://github.com/NVlabs/instant-ngp/assets/111538960/35512f6e-a248-4d7b-a0ef-c1a2ae0481bf)

![Screenshot_1](https://github.com/NVlabs/instant-ngp/assets/111538960/15efb03d-10ea-4ce3-bc88-45e2a294da18)

**Comment by Timothy-pop** [comment]
I have encountered similar issues before.
First, try to minimize the scene to only include the car.
You can start by increasing the Resolution to around 1000.
Then, increase the MC density threshold a bit, say to 4.0.
If you still encounter this issue, feel free to send me the dataset to try.
